msal/src/main/java/com/microsoft/identity/client/IPublicClientApplication.java

```diff
@@ -162,4 +171,39 @@ void acquireToken(@NonNull final Activity activity,
         void onError(final MsalException exception);
     }
 
+    /**
+     * Callback object used in Device Code Flow.
+     * This callback provides the following methods for communicating with the protocol.
+     *                 1). Receiving authentication information (user_code, verification_uri, and instruction message)
+     *                 via {@link DeviceCodeFlowCallback#onUserCodeReceived(String, String, String)}.
```

Comment:
Not quite sure why the last two `String` has underlines here

Reply:
Hmm, not sure, seems to only show up in github

Reply:
I guess it is okay then